spec/controllers/pages_controller_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'spec_helper'
 
-describe PagesController, type: :controller do
+describe PagesController, type: :controller  do
```

Comment:
Hi, @brunoocasali, there is an extra space here. Rubocop is flagging this.

```suggestion
describe PagesController, type: :controller do
```